content-src/asrouter/templates/SubmitFormSnippet/SubmitFormSnippet.jsx

```diff
@@ -150,7 +150,10 @@ export class SubmitFormSnippet extends React.PureComponent {
     return (<SnippetBase {...this.props} className={containerClass} footerDismiss={true}>
         {content.scene2_icon ? <div className="scene2Icon"><img src={content.scene2_icon} /></div> : null}
         <div className="message">
-          <p>{content.scene2_title ? <h3 className="scene2Title">{content.scene2_title}</h3> : null} {content.scene2_text}</p>
+          <p>
+            {content.scene2_title ? <h3 className="scene2Title">{content.scene2_title}</h3> : null}
+            <RichText scene2_text={content.scene2_text} localization_id="scene2_text" />
```

Comment:
Putting this on a new line removes the space between the title and body text which is not desired; perhaps you could add space with `{' '}` to avoid this problem in the future.